pkg/vcs/vcs.go

```diff
@@ -75,6 +75,7 @@ type Commit struct {
 	Author     string
 	AuthorName string
 	CC         []string
+	Recipients []RecipientInfo
```

Comment:
If we have Recipients now, do we still need CC?

Reply:
I think we can remove CC and add them as maintainers using the new struct